support-frontend/assets/helpers/redux/checkout/product/contributionsSideEffects.ts

```diff
@@ -66,20 +66,6 @@ export function addProductSideEffects(
 		},
 	});
 
-	startListening({
-		actionCreator: setProductType,
-		effect(action) {
-			/**
-			 * selectedContributionType is read from storage in 2 places:
-			 * 1) It is used to set the selected contribution type on return
-			 * visits to the Contributions Landing Page in the same session.
-			 * 2) It is used on the Contributions Thank You page to send data to Quantum
```

Comment:
I've tested this on the thank-you page, and it works for all three contribution types.